### ISSUE TITLE
Aggregate function on joined table does not use correct primary key

### ISSUE THREAD

**Issue by davidpurser** [tortoise/tortoise-orm#292]
**Describe the bug**

**To Reproduce**
```
class Groups(Model):
    group_id = fields.IntField(pk=True)

class GroupMemberships(Model):
    group_membership_id = fields.IntField(pk=True)
    item = fields.ForeignKeyField('models.Items')
    group = fields.ForeignKeyField('models.Groups', related_name='group_memberships')

# [...]

    group_member_counts = await Groups.all().annotate(
        member_count=tortoise.functions.Count("group_memberships"))

```

This generates the following query, using the wrong primary key field in the "groups" column (`id` instead of `group_id`):
```
SELECT "groups"."group_id",COUNT("group_memberships"."group_membership_id") "member_count" FROM "groups" LEFT OUTER JOIN "group_memberships" ON "groups"."group_id"="group_memberships"."group_id" GROUP BY "groups"."id"
```

**Expected behavior**
The query generated should be 
```
SELECT "groups"."group_id",COUNT("group_memberships"."group_membership_id") "member_count" FROM "groups" LEFT OUTER JOIN "group_memberships" ON "groups"."group_id"="group_memberships"."group_id" GROUP BY "groups"."group_id"
```

**Additional context**
Line 500 of queryset.py references table.id, which requires that the id field be named 'id'.  Instead, it can be changed to:
```
self.query = self.query.groupby(table.field(self.model._meta.db_pk_field))
```
which will use the correct field name for the database's PK.  However, I'm not 100% sure that's the correct solution.

**Comment by grigi** [comment]
Hi, thank you for this detailed bug report. I'll have a look at it soon. Your guess is very feasible.
Changing it to
```py
self.query = self.query.groupby(table[self.model._meta.db_pk_field])
```
at least doesn't break any existing tests, and seems more correct. I can't confirm it solves the issue until I do some unit tests.

**Comment by grigi** [comment]
Hi, I released v0.15.13 with a fixes related to this. Please test if this resolves your issue :-)
